src/sentry/models/rulesnooze.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+from django.db.models import CheckConstraint, Q
+
+from sentry.db.models import FlexibleForeignKey, Model, region_silo_only_model, sane_repr
+
+
+@region_silo_only_model
+class RuleSnooze(Model):
+    """
+    Duration an issue alert or metric alert is snoozed for a user. Null `until` value means snoozed forever.
+    """
+
+    __include_in_export__ = True
+
+    user = FlexibleForeignKey("sentry.User", unique=True)
```

Comment:
Could we use `HybridCloudForeignKey` here instead? When we move to multi-region schema later this year the user tables will not be adjacent to other organization data.